### ISSUE TITLE
Unwanted black rectangle between fields and alerts in the high contrast on Windows

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#6349]
<img width="652" alt="Screenshot 2024-04-15 at 10 43 26 PM" src="https://github.com/orbeon/orbeon-forms/assets/106961/3a80e4df-a6c3-42ca-a81b-42fb410cbcfe">

[+1 from customer](https://3.basecamp.com/3600924/buckets/1956846/messages/7175826989)

**Comment by ebruchez** [comment]
Solution: we include the arrow only when `prefers-contrast` is either `no-preference` or `less`. The space will be blank instead.

**Comment by ebruchez** [comment]
- [ ] cp on 2022.1-pe
